core-linux-helpers/Dockerfile

```diff
@@ -42,3 +42,12 @@ RUN /tmp/install-platform-packages.sh gcc-multilib lib32stdc++-11-dev
 FROM base AS x86_64
 ENV XTOOLS_HOST=x86_64-linux-gnu
 RUN /tmp/install-platform-packages.sh g++-x86-64-linux-gnu
+
+FROM x-tools-linux-armbe8 as armbe8
+ENV XTOOLS_HOST=armbe8-linux-gnueabi
+
+FROM x-tools-linux-arm64be as arm64be
+ENV XTOOLS_HOST=aarch64_be-linux-gnu
+
+FROM x-tools-linux-arm64beilp32 as arm64beilp32
+ENV XTOOLS_HOST=aarch64_be_ilp32-linux-gnu
```

Comment:
It would be good to re-order (all of) the sections in this file so they follow the order in `.github/workflows/core-linux-helpers.yml`.